index.js

```diff
@@ -1,100 +1,122 @@
+"use strict";
+
 var _ = require('underscore');
+var cnchars = require('cn-chars'); // convert between traditional and simplified
 var path = require('path');
+var pinyin = require('prettify-pinyin'); // prettify the pinyin default (letters + numbers) in CC-CEDICT
 var Sequelize = require('sequelize');
+
 var sequelize = new Sequelize(null, null, null, {
   dialect: 'sqlite',
-  storage: path.join(__dirname, './db/cc-cedict.sqlite')
+  storage: path.join(__dirname, './db/', 'cc-cedict.sqlite')
 });
 
-var Word = sequelize.define('Word', {
```

Comment:
MySQL convention does not have capitalized table names (which this line generates).